AutomatedTesting/Gem/PythonTests/Physics/tests/EntityComponentTests/PhysX_Collider_Component_CRUD.py

```diff
@@ -0,0 +1,167 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+def PhysX_Collider_Component_CRUD():
+    """
+    Summary:
+    Creating a PhysX Collider and setting values to all properties that are supported via the Editor Context Bus.
+
+    Expected Behavior:
+    Validates that Component CRUD can be performed without crashing.
```

Comment:
update: Also should likely say 'and performs expected operations' 😝 Not crashing only seems like a low bar 😉

Reply:
@hultonha updated.

@smurly Yes. That's the standard software engineering expansion of the acronym.